test/e2e-test/nextjs.test.ts

```diff
@@ -72,14 +72,7 @@ describe('Create nextjs application', () => {
     it('init template nextjs"', async () => {
         console.log('Creating Next project');
         const expectOutput = `Export successful. Files written to ${template}/out`
-        await execFile(`npx -y create-next-app@latest ${template}`);
-        const data = fs.readFileSync(`${template}/pages/index.js`);
-        const dataToString = data.toString();
-        const dataToArray = dataToString.split('\n');
-        dataToArray.splice(1,1); //Remove import of NextJS/Image
-        dataToArray.splice(61,1,'               <img src="/vercel.svg" alt="Vercel Logo" width="72" height="16" />\n');
```

Comment:
Interesting ... we do similar things on qa-e2e-tests repo. Does that mean we don't need to do it anymore over there?

Reply:
I based it on qa-e2e repository when I wrote this test, so we have the same problem. This template is simpler and has just common html tags, so it doesn't need to be edited.